packages/ocs/dashboards/persistent-internal/details-card.tsx

```diff
@@ -35,10 +36,13 @@ const DetailsCard: React.FC = () => {
     namespace: CEPH_STORAGE_NAMESPACE,
   });
   const infrastructurePlatform = getInfrastructurePlatform(infrastructure);
-  const cluster = ocsData?.find(
+  const cluster: StorageClusterKind = (ocsData as StorageClusterKind[])?.find(
```

Comment:
use `useK8sList<StorageClusterKind> instead...
```suggestion
  const cluster: StorageClusterKind = ocsData?.find(
```